gst/tensor_filter/tensor_filter.c

```diff
@@ -354,44 +352,37 @@ gst_tensor_filter_fix_caps (GstTensor_Filter * filter, gboolean isInput,
   staticcap = gst_static_caps_get (&rawcap);
 
   if (isInput == TRUE) {
-    type = prop->inputMeta.types;
-    int i;
-    for (i = 0; i < NNS_TENSOR_SIZE_LIMIT; i++) {
-      dimension[i] = prop->inputMeta.dims[i];
-    }
+    meta = &prop->inputMeta;
```

Comment:
You are again doing "shallow copy" while the previous code was doing "deep copy".
Please check if this is safe and if it's safe, constify them.

Reply:
I missed it could be modified. I'll update it